sdk/monitor/monitor-opentelemetry/test/internal/unit/metrics/liveMetrics.test.ts

```diff
@@ -190,9 +190,7 @@ describe("#LiveMetrics", () => {
     const documents = autoCollect.getDocuments();
     assert.strictEqual(documents.length, 16, "documents count");
     // assert.strictEqual(JSON.stringify(documents), "documents count");
-    assert.strictEqual(documents[0].documentType, "Exception");
```

Comment:
Did something changed with the order here or is the exception not present anymore?

Reply:
The first six telemetry items are 1 trace message & 5 exceptions. Before I changed it, the assert was checking for the first document to be an exception (which was never true). I fixed the assert to match the order in which the telemetry was emitted in.